dataactbroker/handlers/aws/session.py

```diff
@@ -152,30 +146,20 @@ def toUnixTime(datetimeValue) :
 
     returns int
     """
-    if(type(datetimeValue) == Decimal):
-        # If argument is already Decimal, assume it's been converted previously
-        return datetimeValue
-    return (datetimeValue-datetime(1970,1,1)).total_seconds()
+    if(type(datetimeValue) == datetime):
+        # If argument is a datetime object, convert to timestamp
+        return (datetimeValue-datetime(1970,1,1)).total_seconds()
+    return datetimeValue
 
 
 
-class DynamoSession(CallbackDict, SessionMixin):
+
+class UserSession(dict, SessionMixin):
```

Comment:
What do you think about putting a `pass` statement here to make it clear that this is intentionally left blank [per the docs](http://flask.pocoo.org/docs/0.11/api/#session-interface)?